src/Modules/Backend/assets/Backend/webpack/scss/components/_buttons.scss

```diff
@@ -84,8 +90,8 @@
 
   &:hover,
   &:focus {
-    background: darken($gray-200, 15%);
-    border-color: darken($gray-200, 15%);
+    background: color.adjust($gray-200, $lightness: -15%);
```

Comment:
why?

Reply:
This is a requirement from stylelint, darken shouldn't be used